packages/wallet-sdk/jest.config.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Should this file be updated to uses the conventions from the other packages, for example https://github.com/blockstack/stacks.js/blob/master/packages/transactions/jest.config.js ?
The existing jest.config.js files are known to work correctly with debuggers.

Reply:
@zone117x Added these changes to follow test conventions from other packages